src/content/blog/2024-06-14-utah-sgid-statewide-roads-data-layer-updates.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Utah SGID Statewide Roads Data Layer Updates
+author: Greg Bunce
+date: 2024-06-14T08:30:00.000Z
+category: SGID Updates
+tags:
+  - roads
+cover_image: /src/images/pillar-blog/road-update-social-card.png
+cover_image_alt: ugrc sgid road update social card
+---
+
+UGRC recently completed updates to the SGID Roads data layer. Please visit our [Roads and Highway System](/products/sgid/transportation/road-centerlines/) data page where you will find information about the Roads data model, as well as a web service layer to the SGID Roads data and direct download links in shapefile and geodatabase format.
+
+These updates are also reflected in UGRC's [address locators](/products/sgid/address/).
+
+The following are highlights from this month's update.
+
+#### County Updates
+
+New roads were added and road names and address ranges were updated for the following counties:
+
+- **Cache County:** Obtained roads data on 05/14/2024. Previous update was on 03/08/2024.
+- **Davis County:** Obtained roads data on 05/08/2024. Previous update was on 03/08/2024.
+- **Iron County:** Obtained roads data on 06/12/2024. Previous update was on 04/15/2024.
+- **Salt Lake County:** Obtained roads data on 05/15/2024. Previous update was on 05/13/2024.
+- **Tooele County:** Obtained roads data on 06/14/2024. Previous update was on 12/15/2024.
+- **Utah County:** Obtained roads data on 05/15/2024. Previous update was on 05/13/2024.
+- **Washington County:** Obtained roads data on 05/15/2024. Previous update was on 05/13/2024.
+- **Wayne County:** Obtained roads data on 04/09/2024. Previous update was on 05/13/2024.
```

Comment:
Is the year wrong for some of these previous update dates? Tooele for sure doesn't look correct to me.

Reply:
good catch. yeah, the Tooele date hadn't arrived yet. thanks